onnxruntime/core/optimizer/reshape_fusion.cc

```diff
@@ -39,9 +39,93 @@ Status ReshapeFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level, c
   return Status::OK();
 }
 
+static bool DistilBertCheck(Graph& graph, const Node& concat, const Node& layernorm_add, const logging::Logger& logger) {
```

Comment:
Could you add some comment about the subgraph pattern? You can refer to comments in Fuse_Subgraph as example.

Reply:
sure